### PR TITLE
Adapt to swift 2.23 (train)

### PR DESCRIPTION
**Pull request by fvennetier**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,26 @@
 sudo: required
-dist: xenial
+dist: bionic
 language: python
-python: '2.7'
+python:
+    - "2.7"
```
Comment (cdelgehier) [inline .travis.yml]: python: 3 ?

File: bin/oioswift-proxy-server [hunk 2]
```diff
@@ -51,7 +56,7 @@ def run_objgraph(types):
     objgraph.show_most_common_types(limit=50, shortnames=False)
     for type_ in types:
         count = objgraph.count(type_)
-        print '%s objects: %d' % (type_, count)
+        print('%s objects: %d' % (type_, count))
```
Comment (cdelgehier) [inline bin/oioswift-proxy-server]: https://docs.python.org/3.0/whatsnew/2.6.html#pep-3101

File: .travis.yml (hunk 1 above)
Comment (fvennetier) [reply .travis.yml]: No. The purpose of this pull request is Openstack Train compatibility. There is a lot more work to do for Python 3 compatibility.

Comment (cdelgehier) [reply .travis.yml]: bitmoji

**Pull request merged**